examples/main.tf

```diff
@@ -241,3 +241,57 @@ resource "instaclustr_cluster" "example-redis" {
     }
   }
 }
+
+resource "instaclustr_cluster" "example_kafka_schema_reg_and_rest_proxy" {
+  cluster_name = "<cluster name>"
+  node_size = "t3.small-20-gp2"
+  data_centre = "US_WEST_2"
+  sla_tier = "NON_PRODUCTION"
+  cluster_network = "192.168.0.0/18"
+  cluster_provider = {
+    name = "AWS_VPC"
+  }
+  rack_allocation = {
+    number_of_racks = 3
+    nodes_per_rack = 1
+  }
+  bundle {
+    bundle = "KAFKA"
+    version = "2.5.1"
+  }
+  bundle {
+    bundle = "KAFKA_REST_PROXY"
+    version = "5.0.0"
+  }
+
+  bundle {
+    bundle = "KAFKA_SCHEMA_REGISTRY"
+    version = "5.0.0"
+  }
+}
+
+data "instaclustr_kafka_schema_registry_user_list" "kafka_schema_reg_user_list" {
+  cluster_id = "${instaclustr_clustr.example_kafka_schema_reg_and_reg_proxy.cluster_id}"
```

Comment:
Doesn't look like `example_kafka_schema_reg_and_reg_proxy` matches the example resource you have defined. Also `instaclustr_clustr`

Reply:
yeah. corrected it.